dev/tasks/tasks.yml

```diff
@@ -801,6 +801,8 @@ tasks:
     ci: github
     template: java-jars/github.yml
     artifacts:
+      - arrow-algorithm-{no_rc_snapshot_version}-cyclonedx.json
+      - arrow-algorithm-{no_rc_snapshot_version}-cyclonedx.xml
```

Comment:
Oh, do we need to repeat this for all jars like `arrow-avro/arrow-c-data/arrow-compression/...`?

Reply:
I think so. That is why @kou said the diff is incomplete for `dev/tasks/tasks.yml` and there is a `# NOTE!!! We need to add more entries for *-cyclonedx.{json,xml}`.